Sources/TuistKit/Generator/WorkspaceGenerator.swift

```diff
@@ -90,15 +90,43 @@ final class WorkspaceGenerator: WorkspaceGenerating {
             workspace.data.children.append(workspaceFileElement(path: relativePath))
         }
 
-        try workspace.write(path: workspacePath.path, override: true)
+        try write(xcworkspace: workspace, to: workspacePath)
 
         return workspacePath
     }
 
+    fileprivate func write(xcworkspace: XCWorkspace, to: AbsolutePath) throws {
```

Comment:
I believe `private` here will do the trick - there aren't any other entities in the source file that need visibility of this method.